Dockerfile

```diff
@@ -1,5 +1,29 @@
-FROM clojure:tools-deps-alpine
-LABEL Name=clojure-representer Version=0.0.1
-COPY . /opt/representer
-WORKDIR /opt/representerer
+FROM babashka/babashka:0.10.163-alpine as builder
+
+# Install SSL ca certificates
+RUN apk update && apk add ca-certificates
+
+# Create appuser
+RUN adduser -D -g '' appuser
+
+# get the source code
+WORKDIR /clojure-representer
+COPY clojure_representer.clj .
+COPY bin/run.sh bin/run.sh
+
+# Get static analyzer image as build stage
+FROM cljkondo/clj-kondo:2021.01.20-alpine as clj-kondo
+
+# Build a minimal and secured container
+FROM babashka/babashka:0.10.163-alpine
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /etc/passwd /etc/passwd
```

Comment:
I assume you're using the builder to not have the overhead of `apk update` within this container? If so, I think there are also options to not have apk update cache or to empty its cache. That way you wouldn't need the builder image. But it's fine to keep it like this of course, just curious.

Reply:
I'm actually curious too, my assumption was that it had to do with security. I just saw that's what everyone else was doing 🐒

Reply:
It could be 🤷